docs/userGuide/syntax/scrollTopButton.md

```diff
@@ -0,0 +1,39 @@
+{% from "userGuide/components/advanced.md" import slot_info_trigger %}
+
+## Scroll To Top Button
+
+<div id="content">
+
+The ```scroll-top-button``` component allows users to move to the top of the page.
+
+****Adding a scroll-top-button****
+
+Add ```<scroll-top-button><scroll-top-button/>``` to layout file.
+
+****Options****
+
+| Name     | Type     | Default                 | Description                                                                           |
+|----------|----------|-------------------------|---------------------------------------------------------------------------------------|
+| icon     | `String` | `fa fa-arrow-circle-up` | Icon used for button.                                                                 |
```

Comment:
Is possible to support something similar to the `icon` syntax in the Box component (e.g. `icon=:fas-rocket:"`)? This may be more user-friendly since it'll be consistent with the Box component and how icons are used in Markbind!